__tests__/app.test.js

```diff
@@ -207,19 +207,20 @@ describe("GET - /api/reviews", () => {
         expect(reviews).toBeInstanceOf(Array);
         expect(reviews).toHaveLength(13);
         expect(reviews).toBeInstanceOf(Array);
+
         reviews.forEach((review) => {
-          expect.objectContaining({
-            review_id: expect.any(Number),
-            title: expect.any(String),
-            review_body: expect.any(String),
-            designer: expect.any(String),
-            review_img_url: expect.any(String),
-            votes: expect.any(Number),
-            category: expect.any(String),
-            owner: expect.any(String),
-            created_at: expect.any(Number),
-            comment_count: expect.any(Number),
-          });
+          expect(review).toEqual(
+            expect.objectContaining({
```

Comment:
object containing will allow the body in the return value and will not fail this test.